docs/GettingStarted.md

```diff
@@ -274,7 +274,7 @@ This uses `bpf_xdp_adjust_head` helper function to encapsulate an outer IP heade
    1. Allow inbound traffic for `xdp_tests.exe` through Windows Defender Firewall. See **Note 1** below.
    2. Run `xdp_tests.exe xdp_encap_reflect_test --remote-ip <IP on the first host>`. See **Note 2** below.
 
-#### Decapsulation Test
+#### De-encapsulation Test
```

Comment:
Please revert.  "Decapsulation" is the industry accepted term here, not de-encapsulation.